server/chat-plugins/chatlog.ts

```diff
@@ -340,7 +543,7 @@ export const pages: PageTable = {
 		if (roomid.startsWith('spl') && roomid !== 'splatoon' && !user.can('rangeban')) {
 			return LogViewer.error("SPL team discussions are super secret.");
 		}
-		if (roomid.startsWith('wcop') && !user.can('rangeban')) {
+		if (roomid.startsWith('wcop') &&	!user.can('rangeban')) {
```

Comment:
Um, the fact that GitHub thinks this was changed worries me.

Weirdly enough, it still passes lint, so I'm confused about what was changed here.

Reply:
lol, apparently it's more obvious here: it's a tab instead of a space.

I'm surprised the linter isn't complaining.